### PR TITLE
Update operator tests

### PR DESCRIPTION
**Pull request by ArielLaub**

### CODE REVIEW

File: test/integration/tests/RegistrationFlow.t.sol [hunk 1]
```diff
@@ -53,16 +59,67 @@ contract RegistrationFlowTest is MockAVSDeployer {
...
+        BN254.G1Point memory pubKey = BN254.hashToG1(keccak256("seed_for_hash"));
+        bytes memory quorumNumbers = BitmapUtils.bitmapToBytesArray(1);
+
+        blsApkRegistry.setBLSPublicKey(operator, pubKey);
+        _setOperatorWeight(operator, 0, 1000);
+
+        ISignatureUtils.SignatureWithSaltAndExpiry memory signature;
+        IEOBLSApkRegistry.PubkeyRegistrationParams memory params;
+        uint96[] memory stakes = new uint96[](1);
+        stakes[0] = 1000;
```
Comment (ArielLaub) [inline test/integration/tests/RegistrationFlow.t.sol:76]: remove it's being read by the stake registry

File: src/EOChainManager.sol [hunk 2]
```diff
@@ -45,53 +56,54 @@ contract EOChainManager is IEOChainManager, OwnableUpgradeable, AccessControlUpg
...
     }
 
     /// @inheritdoc IEOChainManager
-    /// @dev for now there is no state change so the function is view
     function updateOperator(
         address operator,
         uint96[] calldata newStakeWeights
-    ) external view onlyRegistryCoordinator {
+    ) external onlyStakeRegistry {
         // For now just whitelisting. EO chain integration to come.
```
Comment (tafonina) [inline src/EOChainManager.sol]: we have comment  // For now just whitelisting. EO chain integration to come.
but no check for whitelist

File: test/integration/tests/RegistrationFlow.t.sol [hunk 3]
```diff
@@ -53,16 +59,67 @@ contract RegistrationFlowTest is MockAVSDeployer {
...
+        _setOperatorWeight(operator, 0, stakes[0]);
+
+        ISignatureUtils.SignatureWithSaltAndExpiry memory signature;
+        IEOBLSApkRegistry.PubkeyRegistrationParams memory params;
+        vm.expectEmit(true, true, true, true);
+        emit DataValidatorRegistered(operator, stakes);
+        cheats.prank(operator);
+        registryCoordinator.registerOperator(quorumNumbers, params, signature);
```
Comment (Sergei-Liakh) [inline test/integration/tests/RegistrationFlow.t.sol:104]: Lines 91-104 are exact copy from the previous function. Move the code to internal function and reuse it

Comment (ArielLaub) [reply test/integration/tests/RegistrationFlow.t.sol:104]: we have such a function but in these two cases I didn't manage to use an expectEmit call. I had to embed it in the exact place, that's why in these two functions I am not using the shared function

**Review by Sergei-Liakh — APPROVED** [review]

**Review by tafonina — APPROVED** [review]

**Pull request merged**